src/main/java/com/google/sps/servlets/MentorEvidenceServlet.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.classes.SqlConstants;
+import com.google.sps.classes.Utility;
+import java.io.IOException;
+import java.sql.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet that handles mentor evidence in database.
+ */
+@WebServlet("/mentor-evidence")
+public class MentorEvidenceServlet extends HttpServlet {
+  
+  /**
+   * Receives evidence by a mentor to prove them as a previous intern and adds it to database.
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get variable from HTML form.
+    String paragraph = request.getParameter("paragraph");
+
+    // Insert mentor evidence to database.
+    addMentorEvidence(paragraph);
+    response.sendRedirect("/index.html");
+  }
+
+  /**
+   * Inserts evidence provided by mentor to MentorEvidence table in database.
+   */
+  private void addMentorEvidence(String paragraph) {
+    int userId = Utility.getUserId();
+
+    // Set up query to insert new experience tag to user.
+    // Use replace in case mentor evidence already exists in database and mentor wants to update
+    // their information.
```

Comment:
Is this behavior obvious to the mentor?

Reply:
It is not. The reason I did this now is to not have to restrict access to a mentor from entering this page again, and if he inputs information, no errors are caused. I also had in mind that in the future, we could let them know they have this option of updating their info (given we had the time to implement this). I added a TODO, is that ok?

Reply:
I think it's less that they have the option, and more that they could accidentally erase their justification without realizing it. Definitely include that motivation in the TODO, it's fine to leave it as a TODO for now.